app/code/core/Mage/Adminhtml/controllers/Catalog/Product/AttributeController.php

```diff
@@ -179,8 +179,8 @@ protected function _filterPostData($data)
             }
 
             if (!empty($data['option']) && !empty($data['option']['value']) && is_array($data['option']['value'])) {
-                $allowableTags = isset($data['is_html_allowed_on_front']) && $data['is_html_allowed_on_front']
-                    ? sprintf('<%s>', implode('><', $this->_getAllowedTags())) : null;
+                $allowableTags = empty($data['is_html_allowed_on_front'])
```

Comment:
A small question here: Wouldn't it be enough to use just isset here, so remove the second part? Because the defintion of isset is: 
`Determine if a variable is declared and is different than null`

In this case a flip of the ternary operator wouldn't be necessary.

Reply:
For `isset($data['is_html_allowed_on_front']) && $data['is_html_allowed_on_front']`, `isset()` checks if the key exists in the array and  `$data['is_html_allowed_on_front']` is not `null`. The second part checks for [a truthy value](https://developer.mozilla.org/en-US/docs/Glossary/Truthy).  This is the same as `isset($someVar[0]) && $someVar[0];` in my description, which is the same as `!empty($someVar[0])`. This [article](https://phpdelusions.net/articles/empty) gives a good explanation. 

When the `!` is removed in `!empty($someVar[0])`, we need to flip the value.

```php 
if (!empty($someVar[0])) {
    $var = 'not empty';
} else {
    $var = 'empty';
}
$var = !empty($someVar[0]) ? 'not empty' : 'empty';

//it's the same as:
if (empty($someVar[0])) {
    $var = 'empty';
} else {
    $var = 'not empty';
}
$var = empty($someVar[0]) ? 'empty' : 'not empty';
```

Reply:
Yeah I overlooked that the value of `$data['is_html_allowed_on_front']` has to be taken into account, not only that there is something that is not null but that there is something truthy. Therefore to check `empty()` is the right choice. Thanks for your answer.